tests/natural_language/LanguageToolBearTest.py

```diff
@@ -6,13 +6,6 @@
 from tests.BearTestHelper import generate_skip_decorator
 from tests.LocalBearTestHelper import verify_local_bear
 
-try:
```

Comment:
this small block needs to stay here.  See the voodoo in https://github.com/coala/coala-bears/pull/1135/files to work around this problem.

ie. use

```
    import language_check
    import guess_language
    language_check
    guess_language
```